src/spikeinterface/core/binaryfolder.py

```diff
@@ -52,6 +53,7 @@ def __init__(self, folder_path):
         self._bin_kwargs = d["kwargs"]
         if "num_channels" not in self._bin_kwargs:
             assert "num_chan" in self._bin_kwargs, "Cannot find num_channels or num_chan in binary.json"
+            warnings.warn(f"`num_chan` contained in folder {folder_path} is deprecated: it will not be redeable starting with version 0.100.")
```

Comment:
We should maintain back-compatibility even after 0.100!

Reply:
So @DradeAW @h-mayorquin I would remove the deprecation warning at loading time...

Reply:
Removed.